golos.ctrl/include/golos.ctrl/parameters.hpp

```diff
@@ -85,8 +94,9 @@ struct [[eosio::table]] ctrl_state {
     msig_perms_param    msig_perms;
     witness_votes_param witness_votes;
     update_auth_param   update_auth_period;
+    vesting_acc_prm     vesting_acc_param;
```

Comment:
`_param` is used in type name and in field name in this two lines. this is bad naming, causing confusion